pkgs/by-name/ma/maplestory-cursor/package.nix

```
@@ -0,0 +1,53 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenvNoCC,
+
+  # build deps
+  clickgen,
+  python3Packages,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "maplestory-cursor";
+  version = "1.0.0";
```

Version not specified at all, use unstable versioning or assign a version in repo (version file, version as an attribute inside a file, as tag, ...)

```suggestion
  version = "0-unstable-2025-07-09";
```